server/controllers/users.js

```diff
@@ -187,3 +187,39 @@ export function getUserCampaignAssociation(req, res) {
       res.status(404).json({ message: `Cannot process request for campaign_user association: ${err}` });
     });
 }
+
+export function updateUserCallSIDField(req, res, next) {
+  const { id } = req.params;
+  const call_sid = 'CAdksl234591adfide294821kdau3u3933';
+  const params = { id, call_sid };
+  return usersService.updateUserById(params)
+    .then((user) => {
+      if (user) {
+        res.status(200).json(user);
+      } else {
+        next();
```

Comment:
we should return a 404 here and a 500 below in the catch block. same applies for your other method.

Reply:
@perfectlynormalbeast can you help me understand what will make it land in the 404 in the else? I can't seem to make it get there while manually testing.